src/screens/SkipLinks.js

```diff
@@ -31,12 +31,6 @@ export default () => (
             'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
           label: 'Storybook',
         },
-        {
-          url:
-            'https://codesandbox.io/s/github/grommet/grommet-sandbox?initialpath=/skiplinks&module=%2Fsrc%2FSkipLinks.js',
```

Comment:
There is a codesandbox for Skip Links, it is just named Skip Link. The url should be changed to `url: 'https://codesandbox.io/s/github/grommet/grommet-sandbox?initialpath=/skiplinks&module=%2Fsrc%2FSkipLink.js'`

Reply:
Thanks for the catch, added the url in! 🙏